routes/soldiers.js

```diff
@@ -0,0 +1,26 @@
+const soldierDb = require("../collections/soldiers");
+const url = require("url");
+
+function handleRequest(db, method, url, data, callback) {
```

Comment:
you can implement this without having to pass the db in every method call. since the db doesn't change after initialization